app/controllers/api/v2/ansible_roles_controller.rb

```diff
@@ -32,28 +33,46 @@ def destroy
         process_response @ansible_role.destroy
       end
 
-      api :PUT, '/ansible_roles/import', N_('Import Ansible roles')
+      api :PUT, '/ansible_roles/import', N_('DEPRECATED: Import Ansible roles'), deprecated: true
       param :proxy_id, :identifier, :required => true, :desc => N_('Smart Proxy to import from')
-      param :role_names, Array, N_('Ansible role names to import')
+      param :role_names, Array, N_('Ansible role names to imported')
       def import
-        @imported = @importer.import!(role_names)
+        Foreman::Deprecation.api_deprecation_warning(_('Use /ansible_roles/sync to sync role with ansible smart proxy'))
```

Comment:
If this is going to be translated then `/ansible_roles/sync` should be extracted from the string. And probably it should be something like `to sync roles from Smart Proxy with Ansible feature enabled` or similar...